Jenkinsfile

```diff
@@ -22,7 +22,8 @@ pipeline {
                     string(credentialsId: 'bintray_username', variable: 'BINTRAY_USERNAME'),
                     string(credentialsId: 'bintray_api_key', variable: 'BINTRAY_APIKEY')]
             ) {
-              sh "mvn versions:set -DnewVersion=1.0.0-${gitCommit()}"
+              def String timestamp = new Date().format("yyyyMMddHHmmss", TimeZone.getTimeZone('UTC'))
+              sh "mvn versions:set -DnewVersion=${timestamp}"
```

Comment:
technically should be zero-padded to a max length but I don't think that'll be a problem here

Reply:
Also bump the major version to `2.0.0-<timestamp>` so it beats all existing versions

Reply:
Ignore me, we're not going to back to the 1990's so we'll always beat existing versions

Reply:
I'm clearly having a Friday moment and it's only Tuesday. Ignore all my comments and go with the approval.